DESCRIPTION

```diff
@@ -14,6 +14,7 @@ Depends:
 Imports: 
     dplyr,
     eurostat,
+    extrafont,
```

Comment:
Okay thanks. I'll use the vanilla Helvetica font for now then. Since I am using that, I do not need the .onLoad() function anymore. I will keep this PR as a reminder for the future where we would need to have 'Roboto' as a font.

cc' @AnneSchoenauer